continuous_integration/environment-3.7.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
   # sqlalchemy 1.4.0 causes deprecation warnings to be raised from pandas
   # along with other issues https://github.com/pandas-dev/pandas/issues/40467
   - sqlalchemy<1.4.0
-  - pyarrow=0.14
+  - pyarrow=1
```

Comment:
Going to try bumping the `pyarrow` minimum a bit. Not sure if that is an acceptable solution (given as I don't know the motivation for the current pinning and whether that is still relevant). That said, it is worth noting that Arrow is currently 4.0.1. So a lot of versions have come and gone recently.

FWIW RAPIDS is moving to Arrow 4 as well ( https://github.com/rapidsai/cudf/pull/7495 ). Not sure where other community stakeholders are.

Reply:
This seems to ensure we get a new enough version of `sparse`

Reply:
PyArrow 1.0.0 is 1 year old exactly. It feels perfectly reasonable for me to require it.

Reply:
cc @jorisvandenbossche @rjzamora in case either of you have thoughts on bumping our minimum `pyarrow` version to 1.0

Reply:
I would be very happy for the minimum pyarrow version to be 1.0 :) This is not an issue for cudf at all (which is moving to >4 as John already noted)